src/client/pages/Offer/Introduction/Sidebar/CancellationOptions.tsx

```diff
@@ -10,12 +10,12 @@ const HandleSwitchingWrapper = styled.div`
   padding: 0 0.25rem;
 `
 
-const HandleSwitchingLabel = styled.label<{ isClickable: boolean }>`
+const HandleSwitchingLabel = styled.label<{ isEnabled: boolean }>`
   display: flex;
   align-items: center;
 
   &:hover {
-    cursor: ${({ isClickable }) => (isClickable ? 'pointer' : 'initial')};
+    cursor: ${({ isEnabled }) => (isEnabled ? 'pointer' : 'not-allowed')};
```

Comment:
I'm just thinking if the `default` cursor would be enough? Since that's what I think we use for all other disabled (otherwise clickable) elements, and I'm not sure how common it is to use `not-allowed` these days 🤔 

I guess it's clearer with the `not-allowed` but it's also... ugly and like, demands a lot of attention (🙈)? And yes, this second point is obviously mostly personal preference.